boards/nucleo-f401/include/periph_conf.h

```diff
@@ -0,0 +1,426 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_stm32f3discovery
+ * @{
+ *
+ * @file
+ * @name       Peripheral MCU configuration for the STM32F4discovery board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Peter Kietzmann <peter.kietzmann@haw-hamburg.de>
+ */
+
+#ifndef __PERIPH_CONF_H
+#define __PERIPH_CONF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Clock system configuration
+ * @{
+ */
+#define CLOCK_USE_HSI
```

Comment:
I would prefer to set this value to 1 or 0 and use `#if CLOCK_USE_HSI` for checking the status